### PR TITLE
Feature/pmg 184 sidebar improvements

### PR DESCRIPTION
**Pull request by sebasrival**
## Description

Navbar fixed
Sidebar fixed
Improved responsiveness for mobile view

## How Has This Been Tested?

Manual Testing

### Do you have any tests?
- [ ] Yes
- [x] No

### CODE REVIEW

File: front-end/src/app/shared/layout/header/header.component.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export class HeaderComponent implements OnInit {
 
   toggle() {
     this.toggled = !this.toggled;
+    console.log(this.toggled)
```
Comment (mauri-medina) [inline front-end/src/app/shared/layout/header/header.component.ts]: Please remove console.log

File: front-end/src/app/shared/layout/header/header.component.css [hunk 2]
```diff
@@ -54,31 +50,66 @@ body {
...
+
+  .toogled-page {
+    margin-left: 15rem;
+  }
+
+  .toggled-nav {
+    left: 15rem !important;
+  }
+
+  /*Contenido movil*/
```
Comment (mauri-medina) [inline front-end/src/app/shared/layout/header/header.component.css]: Please write comments in English so we can  maintain consistency throughout the project

Comment (mauri-medina) [inline front-end/src/app/shared/layout/header/header.component.css]: Please write comments in English so we can maintain consistency throughout the project

File: front-end/src/app/shared/layout/header/header.component.ts (hunk 1 above)
Comment (mauri-medina) [inline front-end/src/app/shared/layout/header/header.component.ts]: Please remove console.log

**Review by luifer-villalba — DISMISSED** [review]
Besides the comments @mauri-medina did and the changes required before the merge.

To me, code tested and verified.

**Review by mauri-medina — DISMISSED** [review]
Thank you 👍

**Review by luifer-villalba — DISMISSED** [review]
Already approved the code before. Thanks for taking the time and effort to maintain the code clear and legible. Well done!

**Comment by luifer-villalba** [comment]
@joseiba: we need to resolve conflicts before merging the pull request.

**Comment by joseiba** [comment]
> @joseiba: we need to resolve conflicts before merging the pull request.

Okay, it's solved

**Review by luifer-villalba — APPROVED** [review]

**Review by mauri-medina — APPROVED** [review]

**Pull request merged**